### PR TITLE
Primer commit

### PR DESCRIPTION
**Pull request by MaaooVaass**
## Description
create a project in ruby on rails.
## Trello card 
"https://trello.com/c/spisKqEk/15-crear-aplicaci%C3%B3n-rails-wbooksapi"

### CODE REVIEW

**Comment by lmasello** [comment]
@MaaooVaass Next time, try to write commit messages in English :)

**Review by lmasello — CHANGES_REQUESTED** [review]
Add a `.ruby-version` file pointing to the Ruby version we are using (in this case, the content of the file should be just `2.5.3`)

More info: https://github.com/rbenv/rbenv#choosing-the-ruby-version

**Review by lmasello — CHANGES_REQUESTED** [review]
Then, we should follow this guide for Pull Request in Wolox:
https://github.com/Wolox/tech-guides/blob/master/git/docs/pull-requests-good-practices.md

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```
Comment (fedeest) [inline Gemfile]: Here you should change the URL to the URL of this repo.

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+5.2.1
```
Comment (lmasello) [inline .ruby-version]: It should match the version stated in the Gemfile (2.5.3)

**Review by lmasello — CHANGES_REQUESTED** [review]

**Review by lmasello — APPROVED** [review]

**Pull request merged**